### PR TITLE
feat(integrations): add support for BuiltWith

### PR DESCRIPTION
**Pull request by gkhngyk**
What is BuiltWith ?
[BuiltWith®](https://builtwith.com) is a website profiler, lead generation, competitive analysis and business intelligence tool providing technology adoption, ecommerce data and usage analytics for the internet.

BuiltWith technology tracking includes widgets, analytics, frameworks, content management systems, advertisers, content delivery networks, web standards and web servers to name some of the technology categories covered.

### CODE REVIEW

File: packages/shared/providers.yaml [hunk 1]
```diff
@@ -1239,6 +1239,27 @@ buildium:
...
+        base_url: https://api.builtwith.com
+        query:
+            KEY: ${apiKey}
+    docs: https://docs.nango.dev/integrations/all/builtwith
+    credentials:
+        apiKey:
+            type: string
+            title: API Key
+            description: The API key for your BuiltWith account
+            example: 1bc32cba-a5d6-438a-bbcc-af312f560a3c
```
Comment (hassan254-prog) [inline packages/shared/providers.yaml:1262]: You could also add the pattern here, using your example, you could use 
`format: uuid`

Comment (hassan254-prog) [reply packages/shared/providers.yaml:1262]: Also prepared a template logo you could [use](https://codefile.io/f/Px3RIYNx5v)

Comment (hassan254-prog) [reply packages/shared/providers.yaml:1262]: Could you please also add the `connect-ui` documentation? You may refer to the following [example](https://github.com/NangoHQ/nango/pull/3121/files#diff-db1f0aeac4811ed0b6ff9e1b043d645fee90550bbf0b123b70541e8d8ec6cca6).

**Review by hassan254-prog — COMMENTED** [review]
Thanks @gkhngyk for the contribution, small comments inline otherwise LGTM 💪

File: packages/shared/providers.yaml (hunk 1 above)
Comment (gkhngyk) [reply packages/shared/providers.yaml:1261]: @hassan254-prog  Can you help me solve the above problem

**Comment by gkhngyk** [comment]
@hassan254-prog  i solve all issues. Can you check again?

File: docs-v2/integrations/all/builtwith.mdx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+<Tip>Need help getting started? Get help in the [community](https://nango.dev/slack).</Tip>
+
+<Note>
+    Contribute improvements to the setup guide by [editing this page](https://github.com/nangohq/nango/tree/master/docs-v2/integrations/all/builtwith.mdx)
+</Note>
+
+## Useful links
+
+- [Generate a BuiltWith API key in your BuiltWith account](https://api.builtwith.com)
```
Comment (khaliqgant) [inline docs-v2/integrations/all/builtwith.mdx:37]:
```suggestion
- [Generate a BuiltWith API key in your BuiltWith account](https://api.builtwith.com)
- [BuiltWith API Docs](https://api.builtwith.com/free-api)
```

File: packages/shared/providers.yaml [hunk 3]
```diff
@@ -1239,6 +1239,29 @@ buildium:
...
+            KEY: ${apiKey}
+    docs: https://docs.nango.dev/integrations/all/builtwith
+    docs_connect: https://docs.nango.dev/integrations/all/builtwith/connect
+    credentials:
+        apiKey:
+            type: string
+            title: API Key
+            description: The API key for your BuiltWith account
+            example: 1bc32cba-a5d6-438a-bbcc-af312f560a3c
+            format: uuid
```
Comment (khaliqgant) [inline packages/shared/providers.yaml:1263]:
```suggestion
            format: uuid
            doc_section: '#step-1-finding-your-api-key'
```

**Review by khaliqgant — CHANGES_REQUESTED** [review]
Looks great, thanks for your contribution! Couple of small suggestions to make this complete 😀

**Review by khaliqgant — APPROVED** [review]

**Pull request merged**